packages/push/src/instruction/int/negate.rs

```diff
@@ -130,7 +133,7 @@ mod tests {
     // We need to make sure that `x` is greater than `i64::MIN` since
     // we handle that case differently. This is described in the documentation
     // for `Negate`, and handled in the preceding test.
-    fn negate(#[filter(#x > i64::MIN)] x: i64) {
+    fn negate(#[strategy(((i64::MIN+1)..=i64::MAX))] x: i64) {
```

Comment:
```suggestion
    fn negate(#[strategy((i64::MIN+1)..=i64::MAX)] x: i64) {
```

Reply:
And probably for this test as well (the `IntInstruction::foo()` -> `Foo` thing), but we might want to do it in another pr since this isn't actually related to `Abs`

Reply:
Actually, I'm not entirely sure what you mean by `IntInstruction::foo()`, etc., here.